docs/src/pages/[platform]/connected-components/authenticator/configuration/index.page.mdx

```diff
@@ -8,34 +8,9 @@ supportedFrameworks: all
 import { Fragment } from '@/components/Fragment';
 import { Alert, Tabs, TabItem } from '@aws-amplify/ui-react';
 
-## Zero Configuration
```

Comment:
Moved this to the main authenticator page along with the amplify pull instructions